ws/raflow.go

```diff
@@ -14,11 +14,13 @@ import (
 
 // RAFlowJSONData holds the struct for all the parts being involed in rental agreement flow
 type RAFlowJSONData struct {
-	Dates     RADatesFlowData       `json:"dates"`
-	People    []RAPeopleFlowData    `json:"people"`
-	Pets      []RAPetsFlowData      `json:"pets"`
-	Vehicles  []RAVehiclesFlowData  `json:"vehicles"`
-	Rentables []RARentablesFlowData `json:"rentables"`
+	Dates       RADatesFlowData         `json:"dates"`
+	People      []RAPeopleFlowData      `json:"people"`
+	Pets        []RAPetsFlowData        `json:"pets"`
+	Vehicles    []RAVehiclesFlowData    `json:"vehicles"`
+	Rentables   []RARentablesFlowData   `json:"rentables"`
+	ParentChild []RAParentChildFlowData `json:"parentchild"`
```

Comment:
just keep `RAParentChildFlowData` instead of an array, we're not sure what structure goes in there as of now! Same for Tie.